config/routes.rb

```diff
@@ -38,6 +38,10 @@
     end
   end
 
+  constraints(:id => %r{[^/]+}) do
+    get 'cockpit/host_ssh_params/:id', to: 'cockpit#host_ssh_params'
```

Comment:
@mvollmer here is the updated endpoint for the host_ssh_params: no long nested behind `api/v2`